src/docs/swagger.js

```diff
@@ -38,7 +38,7 @@ export default {
     },
     {
       name: 'ratings',
-      description: 'The ratings posted on articles on Author\'s Haven'
+      description: "The ratings posted on articles on Author's Haven"
```

Comment:
Hi, @codeshifu Nice work so far 👍 . I see you're using double quotes here, Kindly change it to single quotes, thanks